api/core/model_runtime/model_providers/siliconflow/siliconflow.yaml

```diff
@@ -19,7 +19,7 @@ configurate_methods:
   - predefined-model
 provider_credential_schema:
   credential_form_schemas:
-    - variable: api_key
+    - variable: siliconflow_api_key
```

Comment:
Since this provider is based on OpenAI Compatible, you need to change the api_key back, and changing the variable will result in a breaking change.